googling.md

```diff
@@ -0,0 +1,22 @@
+# question number one
+  
+  
+  Phrase 1: how can change a typo in commit messages?
+
+  Phrase 2: how can i change a commit messages before I push it to github?
+
+  # question number two
+   Phrase 1: what are the Ruby error  messages with meaning?
```

Comment:
This is quite generic for a Google phrase. Something like `ruby unterminated string meets end of file` will probably get you closer to the relevant results.